dev/PushNotifications/PushNotificationsLongRunningTask/winmain.cpp

```diff
@@ -1,10 +1,65 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-#include <windows.h>
 #include "pch.h"
 
+// include notifications constants file here
+
+#include <NotificationsLongRunningProcess_h.h>
+
+// Temporarily disable C4324 because WRL generates a false (well, irrelevant) warning
+//   'Microsoft::WRL::Details::StaticStorage<Microsoft::WRL::Details::OutOfProcModuleBase<ModuleT>::GenericReleaseNotifier<T>,Microsoft::WRL::Details::StorageInstance::OutOfProcCallbackBuffer1,ModuleT>': structure was padded due to alignment specifier
+#pragma warning(push)
+#pragma warning(disable:4324)
+#include <wrl.h>
+#pragma warning(pop)
+
+#include "platform.h"
+#include "platformfactory.h"
+
+using namespace Microsoft::WRL;
+
 int WINAPI WinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, PSTR /*lpCmdLine*/, int /*nCmdShow*/)
 {
+    THROW_IF_FAILED(::CoInitializeEx(nullptr, COINITBASE_MULTITHREADED));
```

Comment:
THROW macros in WinMain? This returns int right which is technically a win32 status error

Reply:
These should be RETURN macros.